### ISSUE TITLE
Parish's town field should be optional

### ISSUE THREAD

**Issue by ghost** [sfu-dhil/bep#15]
**Describe the bug**
Many parishes are outside of towns, so town should be optional.0

**Comment by JSCraig** [comment]
Yes, the majority of parishes are rural areas.  Think of them as spatial units, covering several thousand acres. Anyone who lived inside these units had to pay tithes - money in kind or coin - to the church.  Many embraced small villages or hamlets. Only a small number embraced market towns of perhaps 5000 inhabitants.  London is the huge exception, but some of the other more sizable towns - Bristol and Norwich - for example encompassed perhaps as many as 10 parishes.

**Comment by JSCraig** [comment]
I've completed entering most of the parish names.  Only one county - Norfolk left to enter.  I think the total number of parish names will come to about 900 or 950.  Would it make more sense to create an additional field called Villages or Hamlets or simply stick with the name of the parish?  In order to complete data entry, I've created general names - Parish in Somerset or Village Gloucestershire.  I'll send some examples.
<img width="1332" alt="Screenshot 2021-01-08 at 21 10 28" src="https://user-images.githubusercontent.com/75693581/104064647-2fc24e80-51f6-11eb-9b10-d01101929d21.png">

<img width="1246" alt="Screenshot 2021-01-08 at 21 10 13" src="https://user-images.githubusercontent.com/75693581/104064542-09041800-51f6-11eb-9745-3e6abc86b5b3.png">
<img width="1372" alt="Screenshot 2021-01-08 at 12 05 42" src="https://user-images.githubusercontent.com/75693581/104064554-0d303580-51f6-11eb-9063-47a56207e0a6.png">
